src/azure-cli-testsdk/azure/cli/testsdk/vcr_test_base.py

```diff
@@ -380,6 +380,10 @@ def _before_record_request(self, request):  # pylint: disable=no-self-use
         # prevents URI mismatch between Python 2 and 3 if request URI has extra / chars
         request.uri = re.sub('//', '/', request.uri)
         request.uri = re.sub('/', '//', request.uri, count=1)
+        # ADLS remove random uuid created
+        request.uri = re.sub(r'&?syncFlag=[A-Z]+', '', request.uri)
+        request.uri = re.sub(r'&?leaseid=[0-9a-z-]+', '', request.uri)
+        request.uri = re.sub(r'&?filesessionid=[0-9a-z-]+', '', request.uri)
```

Comment:
Please note that we're retiring `vcr_test_base` this fix could be temporary. Additionally, the change is placed at the root of the test base, as a result, it may impact other services' tests. Although tests are passing now, this is a potential issue.